index.html

```diff
@@ -235,20 +235,20 @@ <h3>Latest news</h3>
           <!-- News box Start -->
           <div class="news-box">
             <div class="news-box-title">          
-              <span>21/10/2024</span>
+              <span>25/10/2024</span>
             </div>
             <div class="news-box-content">
-	      <p>Uyuni 2024.10 is now only available as an openSUSE Leap 15.6 based containers product (to be deployed into an openSUSE Leap Micro 5.5 host), with support for Ubuntu 24.04, SUSE Linux Micro 6, compressed POS image templates and new ISO-8601 date format for API endpoints! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
+	      <p>With a poll on <a href="https://github.com/uyuni-project/uyuni/discussions/9302">our GitHub discussions</a>, the Uyuni community agreed to <strong>move the Uyuni Community Hours to the last Thursday of the month</strong>, always at 4pm CEST/CET. See you on October 31st for the next session! For the agenda and all the details, please, consult our <a href="https://github.com/uyuni-project/uyuni/wiki/Uyuni-Community-Hours">dedicated wiki page!</a></p>
```

Comment:
```suggestion
	      <p>With a poll on <a href="https://github.com/uyuni-project/uyuni/discussions/9302">our GitHub discussions</a>, the Uyuni community agreed to <strong>move the Uyuni Community Hours to the last Thursday of the month</strong>, always at 16.00 CEST/CET. See you on October 31st for the next session! For the agenda and all the details, please, consult our <a href="https://github.com/uyuni-project/uyuni/wiki/Uyuni-Community-Hours">dedicated wiki page!</a></p>
```

This way we're consistent with the contact page.